pkg/apis/controlplane/v1beta2/types.go

```diff
@@ -325,6 +325,16 @@ type NodeStatsSummary struct {
 	AntreaClusterNetworkPolicies []NetworkPolicyStats `json:"antreaClusterNetworkPolicies,omitempty" protobuf:"bytes,3,rep,name=antreaClusterNetworkPolicies"`
 	// The TrafficStats of Antrea NetworkPolicies collected from the Node.
 	AntreaNetworkPolicies []NetworkPolicyStats `json:"antreaNetworkPolicies,omitempty" protobuf:"bytes,4,rep,name=antreaNetworkPolicies"`
+	// List of PodMulticast statistics collected from the Node.
```

Comment:
I think Multicast stats API should start from version v1alpha1 if you don't change the feature gate version. 

Besides, does this new kind need to add into the function `addConversionFuncs` https://github.com/antrea-io/antrea/blob/main/pkg/apis/controlplane/v1beta2/conversion.go#L28

Reply:
>  I think Multicast stats API should start from version v1alpha1 if you don't change the feature gate version.

I think the feature gate you mentioned is `Multicast:          {Default: false, PreRelease: featuregate.Alpha},`. There are irrelevant in my point of view. The code you  comment is versioned controlplane API, which is considered an internal API.

>   Besides, does this new kind need to add into the function addConversionFuncs https://github.com/antrea-io/antrea/blob/main/pkg/apis/controlplane/v1beta2/conversion.go#L28

I checked the auto-generated func `Convert_v1beta2_MulticastStats_To_controlplane_MulticastStats` and `Convert_controlplane_MulticastStats_To_v1beta2_MulticastStats`, both LGTM. Could you explain why we need add `addConversionFuncs`?  Actually `addConversionFuncs` confuses me because delete the one kind in `addConversionFuncs` doesn't affect the generated code.

Reply:
@tnqn Could you take a look at these questions? Not sure my description was correct.